airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/orchestrator/DbtJobOrchestrator.java

```diff
@@ -76,7 +84,8 @@ public Optional<String> runJob() throws Exception {
                 destinationLauncherConfig.getNormalizationDockerImage())));
 
     log.info("Running dbt worker...");
-    final Path jobRoot = TemporalUtils.getJobRoot(configs.getWorkspaceRoot(), jobRunConfig.getJobId(), jobRunConfig.getAttemptId());
+    final Path jobRoot = TemporalUtils.getJobRoot(configs.getWorkspaceRoot(),
```

Comment:
@colesnodgrass curious, what line wrap is your IDE configured? This looks like a <150 line wrap.

Reply:
It's set to 150, but for a time it was using our [comment length setting of 100](https://github.com/airbytehq/airbyte/pull/18999).  I updated most of the lines that were wrapped early, but missed a few 😞

Reply:
all good. just making sure I'm not going crazy myself